front/pages/api/w/[wId]/data_sources/managed.ts

```diff
@@ -323,11 +323,11 @@ async function handler(
       if (dataSource.vault.isSystem()) {
         const globalVault = await VaultResource.fetchWorkspaceGlobalVault(auth);
 
-        await DataSourceViewResource.createViewInVaultFromDataSourceIncludingAllDocuments(
+        await DataSourceViewResource.createViewInVaultFromDataSource(
```

Comment:
This should include all parents (with `parentsIn` set to `null`).

Reply:
> This should include all parents (with `parentsIn` set to `null`).

I thought the whole idea here is that we don't allow to create a view with parentsIn set to null ? Actually I think we should not create a custom view at all (creating empty is useless (it actually can't be done with the ui)

Reply:
We still need one in the system for each managed data source as we always use the view to browse the content (even in system).

Looking more at this code, it's okay I missed the global vault aspect.

Reply:
We don't have any custom view in system vault - only default (which is now created in resource directly along with the ds). We use the default view in system vault to browse the content in connection management.

Reply:
Can we get rid of this view creation ?

Reply:
Yep seen IRL with @JulesBelveze.

Reply:
We will remove this extra view in a follow up PR.